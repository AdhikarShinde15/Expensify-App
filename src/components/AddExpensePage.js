import React from "react";

const AddExpensePage = () => (
    <div>
        Add Expense PAge
    </div>
);

export default AddExpensePage
