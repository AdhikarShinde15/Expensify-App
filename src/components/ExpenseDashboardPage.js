import React from "react";

const ExpenseDashboardPage = () => (
    <div>This is from Dashboard Page</div>
);

export default ExpenseDashboardPage;