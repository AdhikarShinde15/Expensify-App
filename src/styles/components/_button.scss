.btn {
    background: $blue;
    border: none;
    outline: none;
    display: inline-block;
    padding: $s-size;
    font-weight: 300;
    font-size: $font-size-large;
    color: white;
    line-height: 1;
    text-decoration: none;
}

.btn--link {
    background: none;
}