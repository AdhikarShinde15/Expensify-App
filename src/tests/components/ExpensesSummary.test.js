import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary";

test('Should correctly render ExpenseSummary with 1 expense ',() => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expenseTotal={235}/>)
  expect(wrapper).toMatchSnapshot();
});

test('Should correctly render ExpenseSummary with multiple expense ',() => {
  const wrapper = shallow(<ExpensesSummary expenseCount={21} expenseTotal={23124455}/>)
  expect(wrapper).toMatchSnapshot();
});