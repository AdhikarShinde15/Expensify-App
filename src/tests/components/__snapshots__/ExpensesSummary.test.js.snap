// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render ExpenseSummary with 1 expense  1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        1
      </span>
       
      expense
       totaling 
      <span>
        $2.35
      </span>
       
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="btn"
        to="/create"
      >
        Add Expense
      </Link>
    </div>
  </div>
</div>
`;

exports[`Should correctly render ExpenseSummary with multiple expense  1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        21
      </span>
       
      expenses
       totaling 
      <span>
        $231,244.55
      </span>
       
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="btn"
        to="/create"
      >
        Add Expense
      </Link>
    </div>
  </div>
</div>
`;
